.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
+}
```

Comment:
I think we should not push the `.vscode/settings.json` to the repo. I will suggest adding this folder into your `.gitignore` file.